### ISSUE TITLE
Add package maintainers

### ISSUE THREAD

**Issue by S-S-X** [luarocks/luarocks#1284]
Is there currently any way to add more package maintainers to luarocks.org site so that other people can manage uploaded modules?

It looks like API keys cannot be used because those seem to always impersonate my personal account and cannot be restricted to selected modules.

Only other option I found was linking more accounts but that seems to be similar to API keys and cannot be restricted to selected modules and instead also used to impersonate my personal account.

If there's currently no way to increase bus factor from 1 / single point of failure could it be considered as a new feature?

**Comment by hishamhm** [comment]
@S-S-X No, this is currently not supported. What people have been doing is creating accounts for projects and sharing API keys for those among maintainers.
